homeassistant/components/bring/todo.py

```diff
@@ -0,0 +1,177 @@
+"""Todo platform for the Bring! integration."""
+from __future__ import annotations
+
+from datetime import timedelta
+import logging
+from typing import TYPE_CHECKING
+
+from python_bring_api.exceptions import BringRequestException
+
+from homeassistant import config_entries, core
+from homeassistant.components.todo import (
+    TodoItem,
+    TodoItemStatus,
+    TodoListEntity,
+    TodoListEntityFeature,
+)
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import BringData, BringDataUpdateCoordinator
+
+_LOGGER = logging.getLogger(__name__)
+
+SCAN_INTERVAL = timedelta(seconds=90)
+
+
+async def async_setup_entry(
+    hass: core.HomeAssistant,
+    config_entry: config_entries.ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up the sensor from a config entry created in the integrations UI."""
+    coordinator = hass.data[DOMAIN][config_entry.entry_id]
+
+    unique_id = config_entry.unique_id
+
+    if TYPE_CHECKING:
+        assert unique_id
+
+    async_add_entities(
+        [
+            BringTodoListEntity(
+                coordinator,
+                bringList=bringList,
+                unique_id=unique_id,
+            )
+            for bringList in coordinator.data.values()
+        ],
```

Comment:
```suggestion
        
            BringTodoListEntity(
                coordinator,
                bringList=bringList,
                unique_id=unique_id,
            )
            for bringList in coordinator.data.values()
        
```
requires some reformatting from ruff locally, but saves you a list creation